File: bin/translate.pl
#!/usr/bin/perl -w
#

use strict;
use Rob;
my $rob=new Rob;
use raeseqlib qw/translate_seq/;

my $f=shift || die "fasta file?";
my $fa = $rob->read_fasta($f);
map {print ">$_\n", translate_seq($fa->{$_}), "\n"} keys %$fa;

File: bin/translate.py
import sys
import os
from robseq import translate
from rob import read_fasta

try:
    faf = sys.argv[1]
except:
    sys.exit(sys.argv[0] + " fasta file")



fa = read_fasta(faf)

for i in fa:
    p = translate(fa[i])
    # print(i + "\t" + p)
    print(">" + i + "\n" + p)
